src/commands/synthetics/test.ts

```diff
@@ -7,14 +7,24 @@ import {
   warnIfDeprecatedPollingTimeoutUsed,
 } from './compatibility'
 import {MainReporter, RunTestsCommandConfig, Suite, Test, TriggerConfig} from './interfaces'
+import {DEFAULT_TEST_CONFIG_FILES_GLOB} from './run-tests-command'
 import {getSuites, normalizePublicId} from './utils/public'
 
 export const getTestConfigs = async (
   config: RunTestsCommandConfig,
   reporter: MainReporter,
   suites: Suite[] = []
 ): Promise<TriggerConfig[]> => {
-  const suitesFromFiles = (await Promise.all(config.files.map((glob: string) => getSuites(glob, reporter))))
+  const files = [...config.files]
+
+  // Only auto-discover with the default glob if the user doesn't give any clue what to run.
+  // If they give publicIds (only), they might be in their home directory so we shouldn't auto-discover for performance reasons.
+  // If they give publicIds and files, then only the given publicIds are run and the test config files are applied on them.
```

Comment:
Small nit: since we updated the if underneath we also need to update the comments here